README.md

```diff
@@ -1,2 +1,64 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # leaderboard
-Microvers JS Project
+> The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the [external Leaderboard API ](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)service.
+
+![screenshot](./app_screenshot.png)
+
+For this application the priority is to create a working version of the leaderboard that preserves user input. 
+Here is the list of project milstones:
+
+Milstone 1: basic page structure.
+Milstone 2: send and receive data from API.
+Milstone 3: final touches.
+
+## Built With
+
+- HTML, CSS, JS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] I would invite you to update the `live link` or remove the section for now if you don't have any live link to make the README professional.